lib/roo_on_rails/railties/new_relic.rb

```diff
@@ -0,0 +1,29 @@
+module RooOnRails
+  module Railties
+    class NewRelic < Rails::Railtie
+      initializer 'roo_on_rails.new_relic' do
+        $stderr.puts 'initializer roo_on_rails.new_relic'
+
+        license_key = ENV['NEW_RELIC_LICENSE_KEY']
+
+        if %w[ test development ].exclude?(Rails.env.to_s) and license_key == 'override-me'
+          abort 'Aborting: NEW_RELIC_LICENSE_KEY must be set in production environments'
+        end
+
+        if license_key.nil?
+          abort 'Aborting: NEW_RELIC_LICENSE_KEY is required'
+        end
+
+        path = %w[newrelic.yml config/newrelic.yml].map { |p|
```

Comment:
This was existing code - Dan just moved it around, I believe.
Personally, I would not extract this into a constant unless it's reused, as that lowers readability.